iothub_client/tests/common_e2e/iothubclient_common_e2e.c

```diff
@@ -949,14 +1168,51 @@ static void send_event_test(IOTHUB_PROVISIONED_DEVICE* deviceToUse, IOTHUB_CLIEN
         // cleanup
         destroy_d2c_message_handle(d2cMessage);
     }
+}
+
+static void send_batch_event_test(IOTHUB_PROVISIONED_DEVICE* deviceToUse, IOTHUB_CLIENT_TRANSPORT_PROVIDER protocol)
```

Comment:
This is different enough from rest of tests that I think some basic comments here are worthwhile.  Like we need to use LL to force the batching to occur since we lose control on convenience layer as at least one hint to future maintainers.

Reply:
That's a good suggestion. Done.